### PR TITLE
if pivotal.pem (and webui pair) exists, ensure permissions in default recipe

### PR DESCRIPTION
**Pull request by srenatus**

### CODE REVIEW

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#
+# Author:: Adam Jacob <adam@chef.io>
```
Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb]: :D maybe change this

Comment (srenatus) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/private_keys.rb]: oh, I thought we'd just copy that around. ok, can do.

**Comment by rmoshier** [comment]
LGTM 👍

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb [hunk 2]
```diff
@@ -49,18 +38,9 @@
...
-  content pivotal_key.to_pem.to_s
-  sensitive true
-end
-
 file OmnibusHelper.bootstrap_sentinel_file do
   owner "root"
   group "root"
   mode "0600"
-  content "You've been bootstrapped, punk. Delete me if you feel lucky. Do ya, Punk?"
+  content "You've been bootstrapped, punk. Delete me if you feel lucky. Do ya, punk?"
```
Comment (tylercloke) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb]: Heh maybe we should just update this message in general xD

**Comment by tylercloke** [comment]
I don't think there is a use case for the user wanting to have custom perms on the pivotal key? Cause this would overwrite them. Seems unlikely / like a bad idea. If not, 👍

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb (hunk 2 above)
Comment (srenatus) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb]: Yeah, I've tried making it more meaningful now, including the time it was bootstrapped...

**Comment by srenatus** [comment]
@tylercloke: I don't think this is an issue. You can change the owner using attributes, but you shouldn't have anything less strict than 0600 on it, I guess.

**Comment by srenatus** [comment]
I've had wilson do a build and test run, [only ppc failed](http://wilson.ci.chef.co/view/Chef%20Server%2012/job/chef-server-12-test/290/). ([binaries here](http://artifactory.chef.co/repo/com/getchef/chef-server/12.8.0+20160722072334/))

**Pull request merged**